share/smack/include/smack.h

```diff
@@ -35,6 +35,8 @@ smack_value_t __SMACK_return_value(void);
 // Sugar for __SMACK_init_func_XXX()
 #define __SMACK_INIT(x) void __SMACK_init_func_##x()
 
+void __SMACK_check_memory_access(void*,long int);
```

Comment:
Add a comment just before this briefly saying what this function does, e.g., it is used for generating memory safety checks.